manubot/ai_revision/ai_revision_command.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import shutil
+import tempfile
+from pathlib import Path
+
+
+def cli_process(args):
+    from manubot_ai_editor import models
+    from manubot_ai_editor.editor import ManuscriptEditor
+
+    # set paths for content
+    content_dir = args.content_directory
+    if not content_dir.is_dir():
+        raise SystemExit(
+            f"content directory is not a directory or does not exist: {content_dir}"
+        )
+
+    # set paths for temporary output
+    tmp_dir = Path(tempfile.mkdtemp(suffix="_manubot_ai_revision"))
+    logging.info(f"using temporary directory: {tmp_dir}")
+
+    # create a manuscript editor and model to revise
+    me = ManuscriptEditor(
+        content_dir=content_dir,
+    )
+
+    # instantiate a model
+    module_class = getattr(models, args.model_type)
+
+    if module_class == models.GPT3CompletionModel:
+        model = module_class(
+            title=me.title,
+            keywords=me.keywords,
+        )
+    else:
+        model_kwargs = parse_kwargs(args.model_kwargs)
+
+        model = module_class(
+            **model_kwargs,
+        )
+
+    # revise
+    me.revise_manuscript(tmp_dir, model, debug=True)
+
+    # copy the revised manuscript back to the content folder
+    for f in tmp_dir.glob("*"):
+        shutil.copyfile(f, content_dir / f.name)
+
```

Comment:
Does the code need to delete `tmp_dir` here? https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp says 
> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.

Everything else looks good to me.

Reply:
Linux deletes temporary files on reboot. I guess windows might not delete them by default. I always thought it was the OSes responsibility to cleanup the temporary directories but I might be wrong.

Reply:
It should not be a problem. The folder is in the system's temporary directory, which will be deleted at some point. And `.md` files there are expected to be small, so there should be no concerns about disk usage.

If we want to delete it, we could put the code inside a `try` block and ignore any errors.

Reply:
Testing on Windows, my default temp directory is
```
>>> tempfile.gettempdir()
'C:\\Users\\agitter\\AppData\\Local\\Temp'
```

I see files there from 2021, which is consistent with discussion [here](https://superuser.com/questions/571901/is-it-safe-to-delete-appdata-local-temp) about running the Disk Cleanup utility to delete temporary files.

On the other hand, the COVID-19 review is the longest Manubot manuscript. Its cumulative Markdown content is still only 1.1. MB, so leaving copies of that in the temp directory isn't going to have any practical consequence. I'm fine leaving it as is.